_posts/how_API_works.md

```diff
@@ -0,0 +1,714 @@
+The Blogger API enables you to integrate Blogger content with your application by using the REST APIs. Before you begin, you will need to set up authorization.
```

Comment:
can you please edit the blog as per contributing guidelines mentioned [here](https://github.com/Mentro-Org/Mentro-Community-Blog/blob/main/contributing.md)